atomicfu/src/commonTest/kotlin/kotlinx/atomicfu/test/DelegatedPropertiesTest.kt

```diff
@@ -0,0 +1,103 @@
+@file:Suppress("INVISIBLE_REFERENCE", "INVISIBLE_MEMBER")
+
+package kotlinx.atomicfu.test
+
+import kotlinx.atomicfu.atomic
+import kotlin.test.*
+
+class DelegatedProperties {
+
+    private val _a = atomic(42)
+    var a: Int by _a
+
+    private val _l = atomic(55555555555)
+    var l: Long by _l
+
+    private val _b = atomic(false)
+    var b: Boolean by _b
+
+    private val _ref = atomic(A(B(77)))
+    var ref: A by _ref
```

Comment:
Please, also add a separate test with `val ref by atomic(...)`. That is, without `_ref` variable.